dnf-behave-tests/features/check-dependencies.feature

```diff
@@ -34,3 +33,31 @@ Examples:
         | --duplicates       |
         | --obsoleted        |
         | --provides         |
+
+
+@bz1543449
+Scenario: Removed scriptlet pre and post dependencies are not reported as missing
+  Given I use repository "dnf-ci-check-dependencies"
```

Comment:
AFAIK the names of repositories that belong to single feature files so far don't have the "dnf-ci-" prefix, so let's keep it consistent? (and name the repository just `check-dependencies`?)

On a side note, I'm still worried it's becoming a bit of a mess, that's why I wanted to agree on some rules a while ago...

Reply:
I see, I have updated it to `check-dependencies`.

I agree, it confused me at least.